build-system/tasks/compile.js

```diff
@@ -424,14 +425,19 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
       process.exit(1);
     };
 
+    const stream = gulp.src(srcs, {base: '.'})
+        .pipe(gulpIf(
+            isThirdPartyFile,
```

Comment:
It appears that this param can be a file regex.

Reply:
You're right. Done in b161b60834e7c7e5002b48555ca5be498fe63ee7.